website/docs/datalevellinking.md

```diff
@@ -0,0 +1,20 @@
+Enables linker optimizations to remove unused data by emitting each data item in a separate section.
```

Comment:
```suggestion
Emit each data item in a separate section. This help linker optimizations to remove unused data.
```
It look like `-ffunction-sections`/`-fdata-sections` gcc flags.

Reply:
Yes, you are correct that these map to -ffunction-sections/-fdata-sections. I have been looking to add the flags in the way you describe, and realized the default is to turn these on. The only reason I was adding this is that in the old module there was a remnant of this option and sounded useful, but given it's on by default, does it still make sense to have it? Is there any reason to turn it off?

Reply:
https://stackoverflow.com/questions/4274804/query-on-ffunction-section-fdata-sections-options-of-gcc
There are some possible drawbacks

Reply:
I've gone ahead and added that last one, let me know

Reply:
Last one seems better, but that might probably go in another PR.